ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/Constants.java

```diff
@@ -213,13 +180,76 @@ public String toString() {
         // TODO: remove this after fixing https://github.com/ballerina-platform/ballerina-standard-library/issues/4245
         HTTP_CODES = Collections.unmodifiableMap(httpCodeMap);
     }
+    public static final Map<String, String> HTTP_CODE_DESCRIPTIONS;
```

Comment:
This could also be an enum right ?
```java
public enum HttpStatusCode {
  STATUS_100(100, "Continue");
  ...

  private final int code;
  private final String description;
}
```

Reply:
The use case here is to get the description for a given status code. That's why I have used a map here. This can not be done with a enum right?

Reply:
Yep, +1 for this approach.